.changeset/curly-peas-peel.md

```diff
@@ -1,5 +1,5 @@
 ---
-'@itwin/itwinui-react': patch
+'@itwin/itwinui-react': minor
```

Comment:
lets stick with patch, we want to avoid minor releases for a little longer

Reply:
Sounds good, changed (https://github.com/iTwin/iTwinUI/pull/1660/commits/084ceed710dc148853e838e2f950eaa5a67901e5)

Reply:
Actually, since both changeset points are patches, should I combine them into one changeset file?

Reply:
two changesets is totally fine